### ISSUE TITLE
spark_apply set .libPaths

### ISSUE THREAD

**Issue by mattpollock** [rstudio/sparklyr#1530]
I know that a lot of work has gone into making packages available to R workers living in spark executors. However one option that does not seem to be supported is simply setting `.libPaths` to something.

Naturally users _could_ do
```
tbl %>%
  spark_apply(function(x, context){.libPaths("something"), ...})
```
but it would be nice to enable a system level configuration (think RStudio Server running on a cluster edge node). Would it be possible to support something similar to `spark.r.command` (e.g., `spark.r.libpaths`) which, if set, will cause `.libPaths(config$spark.r.libpaths)` to be executed before `f` in the workers.

**Comment by mattpollock** [comment]
Maybe using `sparklyr.apply.options.rscript.before` is the right approach here. I can't find documentation on how to use that though. Should the value of this param contain R code to be executed or should it point to a file? If it is a file path then where should the file live (e.g., does `~/before.R` refer to my local home dir or my /user/me dir on HDFS)?

**Comment by javierluraschi** [comment]
@mattpollock that sounds reasonable, here is a PR: https://github.com/rstudio/sparklyr/pull/1533

I haven't tested this at all, would you mind giving it a shot? You can install with:

```r
devtools::install_github("rstudio/sparklyr", ref = "bugfix/spark-apply-rlibpaths")
```

**Comment by mattpollock** [comment]
Thanks @javierluraschi! I'll take a look.

**Comment by mattpollock** [comment]
resolved with #1533
